### PR TITLE
Zeindeer fix nightvision and spawn date

### PR DESCRIPTION
**Pull request by KingApep**
#### Summary
 Bugfixes "Fix Zeindeer spawn date and balance stats to be inline with other zed-wildlife"

#### Purpose of change

From the initial implementation PR Zeindeer were intended to spawn in Autumn/Winter but currently spawn on day 3. This changes the date from (72) to (180).

This change also brings the zombification of deer -> zeindeer more in line with out zombified creatures, namely bears -> zombears.

This changes Zeindeer night vision to be lower than that of base Deers rather than increasing their Night vision (Base deer 12), (Zeindeer NV changed from 16 -> 8).
This is inline with bears having reduced night vision from 8 -> 5 when zombified to Zombears.
HP of Zeindeers is also changed from 200 to 120, which is still a doubled HP from base 60hp deers.
This is inline with bears hp going from 100 - 180 when zombified to Zombears.

Edit: I now realise it is Reindeer -> Zeindeer as opposed to deer but the bugfix and balance changes remain justifiable as argued in my comment below.

#### Describe the solution

The change fixes the starting date without changing pack size/frequency and balances Zeindeer stats to be more inline with other zombified wildlife.

#### Describe alternatives you've considered

Keep the starting date as it is prior to the fix but mainline the PR stat changes.

change the starting date but maintain the current Zeindeer stats.

#### Testing

Changes ongoing

### CODE REVIEW

**Comment by actual-nh** [comment]
Umm... zeindeer aren't zombified deer.

**Comment by anothersimulacrum** [comment]
Someone did a bad job of adding labels :)

**Comment by KingApep** [comment]
Yeah I have no idea about labels, second contribution.
Good call on the Reindeer front; however, the issue of NV increasing versus decreasing from zombification still exists (Reindeer 13-16 as opposed to (13-10/8) and I still believe the spawn date change to be additionally worthwhile.
 The HP gain could potentially be increased however these monsters spawn in frquencies from 3-5 so I believe an HP increase of 120 -> 160 from zombification of reindeer -> Zeindeer could still be more apropriate than its current stats.
For reference, a zombear has 180 HP but of course only spawns in individual frequencies.

**Comment by actual-nh** [comment]
> Yeah I have no idea about labels, second contribution.

No problem - was to anothersimulacrum, not you, actually.

> Good call on the Reindeer front; however, the issue of NV increasing versus decreasing from zombification still exists (Reindeer 13-16 as opposed to (13-10/8) and I still believe the spawn date change to be additionally worthwhile.

Spawn date, definitely (until there's something to specify season instead/as well). How about swapping it so Reindeer have NV 16 and the zeindeer 13? While I am not aware of any studies on reindeer night vision (will check if I get a moment), anything with a range including near/within the arctic circle should probably have decent night vision given how dark it gets for prolonged periods of time (admittedly, these are also ones that are migrating south for the winter...).

> The HP gain could potentially be increased however these monsters spawn in frquencies from 3-5 so I believe an HP increase of 120 -> 160 from zombification of reindeer -> Zeindeer could still be more apropriate than its current stats.
> For reference, a zombear has 180 HP but of course only spawns in individual frequencies.

Reasonable.

**Comment by actual-nh** [comment]
> Spawn date, definitely (until there's some way to specify season instead/as well). How about swapping it so Reindeer have NV 16 and the zeindeer 13? While I am not aware of any studies on reindeer night vision (will check if I get a moment), anything with a range including near/within the arctic circle should probably have decent night vision given how dark it gets for prolonged periods of time (admittedly, these are also ones that are migrating south for the winter...).

As it turns out, they do have good night vision - see [this article](https://royalsociety.org/news/2013/arctic-reindeer-eyesight-adaption/).

**Comment by KingApep** [comment]
My issue is more the fact that for 90% of characters these things can see you well beyond your vision range and hit like a tank.
13 NV is essentially the same a zombie predator so personally whilst 8 might be a bit low for zombified reindeer, none of the other zombie wildlife has similar NV except for zeer, which I believe were added at a similar time.

Personally I'd like to nuke it to 10 but I wouldn't think anything beyond 12 would be healthy due to their speed and quantity frquency. A member of the discord told me his story of being chased by 5 of them smashing through walls able to far outsee him in vision and I agree. I personally dont think they should outvision a player with the night vision cbm

**Comment by actual-nh** [comment]
Understand. Usual night vision for unaugumented characters is, what, 10? With goggles/CBM, something like 20? I think 10 for the zeindeer and 16 for reindeer is reasonable, then.

**Comment by Theawesomeboophis** [comment]
Thank you for fixing this and bringing these issues to my
attention, Regular reindeer do have intentionally high night vision as they
do in real life, however it does make since for zeindeer to have lower
night vision due to decomposing of the eyes during zombification.

**Comment by KingApep** [comment]
Nerfed HP less and shifted NV change to 10 as per suggestion

File: data/json/monstergroups/wilderness.json [hunk 1]
```diff
@@ -156,7 +156,7 @@
       },
       { "monster": "mon_deer", "freq": 4, "cost_multiplier": 2, "pack_size": [ 1, 5 ] },
       { "monster": "mon_reindeer", "freq": 3, "cost_multiplier": 3, "pack_size": [ 2, 6 ] },
```
Comment (actual-nh) [inline data/json/monstergroups/wilderness.json:158]:
```suggestion
      { "monster": "mon_reindeer", "freq": 3, "cost_multiplier": 3, "pack_size": [ 2, 6 ], "conditions": [ "AUTUMN", "WINTER" ] },
```

File: data/json/monstergroups/wilderness.json [hunk 2]
```diff
@@ -156,7 +156,7 @@
...
-      { "monster": "mon_zeindeer", "freq": 4, "cost_multiplier": 10, "pack_size": [ 3, 5 ], "starts": 73 },
+      { "monster": "mon_zeindeer", "freq": 4, "cost_multiplier": 10, "pack_size": [ 3, 5 ], "starts": 180 },
```
Comment (actual-nh) [inline data/json/monstergroups/wilderness.json:159]:
```suggestion
      { "monster": "mon_zeindeer", "freq": 1, "cost_multiplier": 15, "pack_size": [ 2, 5 ], "starts": 180 },
      { "monster": "mon_zeindeer", "freq": 4, "cost_multiplier": 10, "pack_size":[ 3, 5 ], "starts": 180, "conditions": [ "AUTUMN", "WINTER" ] },
```

**Review by actual-nh — COMMENTED** [review]
Just realized (remembered?) that _can_ actually adjust by season...

**Comment by KingApep** [comment]
> Just realized (remembered?) that _can_ actually adjust by season...

Didn't even realise those tags existed, good call

File: data/json/monstergroups/wilderness.json [hunk 3]
```diff
@@ -155,8 +155,9 @@
         "conditions": [ "DAWN", "DUSK", "DAY" ]
       },
       { "monster": "mon_deer", "freq": 4, "cost_multiplier": 2, "pack_size": [ 1, 5 ] },
-      { "monster": "mon_reindeer", "freq": 3, "cost_multiplier": 3, "pack_size": [ 2, 6 ] },
-      { "monster": "mon_zeindeer", "freq": 4, "cost_multiplier": 10, "pack_size": [ 3, 5 ], "starts": 73 },
+      { "monster": "mon_reindeer", "freq": 3, "cost_multiplier": 3, "pack_size": [ 2, 6 ], "conditions": [ "AUTUMN", "WINTER" ] },
```
Comment (actual-nh) [inline data/json/monstergroups/wilderness.json:158]:
```suggestion
      { "monster": "mon_reindeer",
        "freq": 3,
        "cost_multiplier": 3,
        "pack_size": [ 2, 6 ],
        "conditions": [ "AUTUMN", "WINTER" ]
      },
```
This still may not be quite right on the spacing!

File: data/json/monstergroups/wilderness.json [hunk 4]
```diff
@@ -155,8 +155,9 @@
...
+      { "monster": "mon_zeindeer", "freq": 1, "cost_multiplier": 15, "pack_size": [ 2, 5 ], "starts": 180 },
+      { "monster": "mon_zeindeer", "freq": 4, "cost_multiplier": 10, "pack_size":[ 3, 5 ], "starts": 180, "conditions": [ "AUTUMN", "WINTER" ] },
```
Comment (actual-nh) [inline data/json/monstergroups/wilderness.json:160]:
```suggestion
      {
        "monster": "mon_zeindeer",
        "freq": 4,
        "cost_multiplier": 10,
        "pack_size":[ 3, 5 ],
        "starts": 180,
        "conditions": [ "AUTUMN", "WINTER" ]
      },
```

**Comment by KingApep** [comment]
Committed your change suggestions but it seems they've still got errors

**Comment by actual-nh** [comment]
> Committed your change suggestions but it seems they've still got errors

It's complaining about invisible changes in whitespace again; joyful. I just got back from work; if you want to run it through the [online linter](https://dev.narc.ro/cataclysm/format.html), that's fine, or I'll try once I'm up to it.

File: data/json/monstergroups/wilderness.json [hunk 5]
```diff
@@ -155,8 +155,22 @@
...
+      },
+      { "monster": "mon_zeindeer", "freq": 1, "cost_multiplier": 15, "pack_size": [ 2, 5 ], "starts": 180 },
+      {
+        "monster": "mon_zeindeer",
+        "freq": 4,
+        "cost_multiplier": 10,
+        "pack_size":[ 3, 5 ],
+        "starts": 180,
+        "conditions": [ "AUTUMN", "WINTER" ]
+      },
```
Comment (actual-nh) [inline data/json/monstergroups/wilderness.json:173]:
```suggestion
      {
        "monster": "mon_reindeer",
        "freq": 3,
        "cost_multiplier": 3,
        "pack_size": [ 2, 6 ],
        "conditions": [ "AUTUMN", "WINTER" ]
      },
      { "monster": "mon_zeindeer", "freq": 1, "cost_multiplier": 15, "pack_size": [ 2, 5 ], "starts": 180 },
      {
        "monster": "mon_zeindeer",
        "freq": 4,
        "cost_multiplier": 10,
        "pack_size": [ 3, 5 ],
        "starts": 180,
        "conditions": [ "AUTUMN", "WINTER" ]
      },
```
If this doesn't work, some downloads/uploads with a JSON linter run in-between will be needed...

**Pull request merged**